packages/eas-cli/src/commands/env/list.ts

```diff
@@ -0,0 +1,78 @@
+import chalk from 'chalk';
+import dateFormat from 'dateformat';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import {
+  EASEnvironmentFlag,
+  EASVariableFormatFlag,
+  EASVariableScopeFlag,
+} from '../../commandUtils/flags';
+import { EnvironmentVariableFragment, EnvironmentVariableScope } from '../../graphql/generated';
+import { EnvironmentVariablesQuery } from '../../graphql/queries/EnvironmentVariablesQuery';
+import Log from '../../log';
+import formatFields from '../../utils/formatFields';
+import { promptVariableEnvironmentAsync } from '../../utils/prompts';
+
+export default class EnvironmentValueList extends EasCommand {
+  static override description = 'list environment variables for the current project';
+
+  static override hidden = true;
+
+  static override contextDefinition = {
+    ...this.ContextOptions.ProjectConfig,
+    ...this.ContextOptions.LoggedIn,
+  };
+
+  static override flags = {
+    ...EASVariableFormatFlag,
+    ...EASVariableScopeFlag,
+    ...EASEnvironmentFlag,
+  };
+
+  async runAsync(): Promise<void> {
+    let {
+      flags: { environment, format, scope },
+    } = await this.parse(EnvironmentValueList);
+    const {
+      privateProjectConfig: { projectId },
+      loggedIn: { graphqlClient },
+    } = await this.getContextAsync(EnvironmentValueList, {
+      nonInteractive: true,
+    });
+
+    if (scope === EnvironmentVariableScope.Project && !environment) {
+      environment = await promptVariableEnvironmentAsync(false);
+    }
+
+    const variables =
+      scope === EnvironmentVariableScope.Project && environment
+        ? (await EnvironmentVariablesQuery.byAppIdAsync(graphqlClient, projectId, environment))
+            .appVariables
+        : await EnvironmentVariablesQuery.sharedAsync(graphqlClient, projectId);
+
+    if (format === 'short') {
+      for (const variable of variables) {
+        Log.log(chalk`{bold ${variable.name}}=${variable.value || '*****'}`);
```

Comment:
```suggestion
        Log.log(`${chalk.bold(variable.name)}=${variable.value || '***** (This is a secret env variable that can only be accessed on EAS builder and can't be read in any UI. Learn more.)'}`);
```

`Learn more` would need to link to the docs that we don't have at the moment